client/job.lua

```diff
@@ -427,11 +427,59 @@ CreateThread(function()
                 end
             end
 
+            for k, v in pairs(Config.Locations["main2"]) do
+                local dist = #(pos - v)
+                if dist < 1.5 then
+                    sleep = 7
```

Comment:
Why sleep is `7`, should be at `0`

Reply:
did not change anything here just copy paste then made some edits.